pom.xml

```diff
@@ -152,7 +152,7 @@
             <groupId>io.netty</groupId>
             <artifactId>netty-transport-native-kqueue</artifactId>
             <version>${netty.version}</version>
-            <classifier>osx-aarch_64</classifier>
+            <classifier>osx-x86_64</classifier>
```

Comment:
Currently for anyone with new M3s the classifier is osx_x86_64. For others its osx-aarch_64 and for intel chips this dependency itself is not present. This is something we need to really look at and see how we can make this dynamic.

Reply:
Did a quick search, it seems `os-maven-plugin` with `${os.detected.classifier}` works.

```
<project>
  ...
  <build>
    <plugins>
      <plugin>
        <groupId>kr.motd.maven</groupId>
        <artifactId>os-maven-plugin</artifactId>
        <version>1.7.0</version>
        <executions>
          <execution>
            <goals>
              <goal>detect</goal>
            </goals>
            <phase>initialize</phase>
          </execution>
        </executions>
      </plugin>
      ...
    </plugins>
  </build>

  <dependencies>
    <dependency>
      <groupId>your.group</groupId>
      <artifactId>your-artifact</artifactId>
      <version>your-version</version>
      <classifier>${os.detected.classifier}</classifier>
    </dependency>
    ...
  </dependencies>
  ...
</project>
```

We can test it out.

Reply:
I think this will break the Intel chip version as the dependency is not available for them so the compile would fail. Il revert the changes for now and see if there is an elegant solution in the future.